packages/kit/src/provider/Container/InAppNotification/index.tsx

```
@@ -34,6 +57,178 @@ const InAppNotification = () => {
+            setTimeout(() => {
+              approvingSuccessActionConfirm();
+            }, 50);
+          } else {
+            // 1.swap tab no modal
+            // 不用做任何动作，直接给 swap 发 event 进行询价
+            if (swapApprovingTransaction) {
+              appEventBus.emit(EAppEventBusNames.SwapApprovingSuccess, {
+                approvedSwapInfo: swapApprovingTransaction,
+                enableFilled: true,
+              });
+            }
```

ESLint

[error] Unexpected if as the only statement in an else block.

(no-lonely-if)